packages/connectivity/src/scp-cf/jwt.ts

```diff
@@ -15,6 +15,124 @@ const logger = createLogger({
   messageContext: 'jwt'
 });
 
+/**
+ * Get the user ID from the JWT payload.
+ * @param jwtPayload - Token payload to read the user ID from.
+ * @returns The user ID, if available.
+ * @internal
+ */
+export function userId({ user_id }: JwtPayload): string {
+  return user_id;
+}
+
+/**
+ * Get the user name from the JWT payload.
+ * @param jwtPayload - Token payload to read the user name from.
+ * @returns The user name if available.
+ * @internal
+ */
+export function userName({ user_name }: JwtPayload): string {
+  return user_name;
+}
+
+/**
+ * Get the user's given name from the JWT payload.
+ * @param jwtPayload - Token payload to read the user's given name from.
+ * @returns The user's given name if available.
+ * @internal
+ */
+export function userGivenName({ given_name }: JwtPayload): string {
+  return given_name;
+}
+
+/**
+ * Get the user's family name from the JWT payload.
+ * @param jwtPayload - Token payload to read the user's family from.
+ * @returns The user's family name if available.
+ * @internal
+ */
+export function userFamilyName({ family_name }: JwtPayload): string {
+  return family_name;
+}
+
+/**
+ * Get the user's e-mail address from the JWT payload.
+ * @param jwtPayload - Token payload to read the user e-mail address from.
+ * @returns The user's e-mail address if available.
+ * @internal
+ */
+export function userEmail({ email }: JwtPayload): string {
+  return email;
+}
+
+/**
+ * Get the user's scopes from the JWT payload.
+ * @param jwtPayload - Token payload to read the user's scopes from.
+ * @returns The user's scopes if available.
+ * @internal
+ */
+export function userScopes(jwtPayload: JwtPayload): string[] {
+  const scopes = jwtPayload.scope;
+  if (Array.isArray(scopes)) {
+    return scopes.map(scope =>
+      scope.includes('.')
+        ? scope.substr(scope.indexOf('.') + 1, scope.length)
+        : scope
+    );
+  }
+  return [];
+}
+
+/**
+ * Extracts the custom attributes from the JWT.
+ * @param payload - Token payload to read the custom attributes from.
+ * @returns Custom attributes added by the XSUAA service to the issued JWT.
+ * @internal
+ */
+export function customAttributes(payload: JwtPayload): Record<string, any> {
```

Comment:
As discussed, perhaps move these into their own object so that a user can access all properties from one location with nice code completion in their IDE.
Other than that LGTM